frontends/api/src/clients.ts

```diff
@@ -55,6 +56,8 @@ const widgetListsApi = new WidgetListsApi(undefined, BASE_PATH, axiosInstance)
 
 const usersApi = new UsersApi(undefined, BASE_PATH, axiosInstance)
 
+const schoolsApi = new SchoolsApi(undefined, BASE_PATH, axiosInstance)
```

Comment:
We should fix this (not necessarily in this PR) for sailing very close to the "required parameter cannot follow an optional" rule. Perhaps a limitation of the generated API client.